### PR TITLE
doc: add company user cases(Tencent, Meituan, Huawei...)

### PR DESCRIPTION
**Pull request by TommyLemon**
## What type of PR is this?
- [ ] bug
- [ ] feature
- [x] enhancement

## How do you solve it?
Add company user cases

## Special notes for your reviewer, ex. impact of this fix, design document, etc:
None

## Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [x] N/A

Affects:
- [x] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

doc: add company user cases(Tencent, Meituan, Huawei...)

### CODE REVIEW

File: README-CN.md [hunk 1]
```diff
@@ -29,7 +29,9 @@
 
 
 
-**NebulaGraph** 是一款开源的图数据库，擅长处理千亿个顶点和万亿条边的超大规模数据集。
+**NebulaGraph** 是一款开源的图数据库，擅长处理千亿个顶点和万亿条边的超大规模数据集。 <br />
+已被腾讯、华为、美团、中国移动、网易游戏、快手、携程、OPPO、自如、360 等众多大厂和小红书、朴朴、知乎等大量中小企业使用。
+![image](https://user-images.githubusercontent.com/5738175/186084533-cb9086cd-1914-4b9f-9a11-f34190479b55.png)
```
Comment (wey-gu) [inline README-CN.md:34]: - The picture looks not polished
- It's not dark-mode aware

Comment (TommyLemon) [reply README-CN.md:34]: I captured it from the 2 pages of the official website:

https://nebula-graph.io/cases
https://nebula-graph.com.cn/cases

Can you provide better pictures?

Comment (TommyLemon) [reply README-CN.md]: Do you think it better? I changed the color of the background to pure white.
image

https://github.com/vesoft-inc/nebula/blob/TommyLemon-patch-2/README-CN.md

**Comment by wey-gu** [comment]
Try something like this instead, it's dark-mode friendly.

https://user-images.githubusercontent.com/1651790/186096704-0bc2cb80-cc40-4aa2-901c-1d493b3f6d30.png

File: README-CN.md (hunk 1 above)
Comment (wey-gu) [reply README-CN.md]: It's better in the light mode now, but not in dark mode :-p

**Pull request closed without merging**